### ISSUE TITLE
Episode does not render in SwiftUI

### ISSUE THREAD

**Issue by palmerhyde** [joreilly/MortyComposeKMM#27]
Hi John. I created a pull request for this but do not have permissions. If you grant me permissions I will try again.

in SwiftUI the episode number does not render. This is because episode.episode is in quotes. The fix is straightforward. In EpisodesListRowView.swift on line 13:

change ```Text("episode.episode")```  to ```Text(episode.episode)```

**Comment by joreilly** [comment]
Hi, I don't think I should have to grant permission to open PR...where are you seeing that that's the case? I'll double check if any settings need to be updated for you repo

**Comment by joreilly** [comment]
Hmm, a few other folks have created PRs for this repo

**Comment by joreilly** [comment]
We're you able to create fork?

**Comment by palmerhyde** [comment]
I think it is a permissions issue on my side. I am investigating

**Comment by palmerhyde** [comment]
I am able to create a fork.

**Comment by palmerhyde** [comment]
Yeah, there is something wrong with the permissions on my end. I was able to create a fork, edit within the fork and then create a pull request from my fork into your repo.
